cmd/util.go

```diff
@@ -0,0 +1,10 @@
+package cmd
+
+import "github.com/pkg/errors"
+
+func ifFilesPassed(files []string) error {
```

Comment:
Name of this function sounds a bit strange, but that is probably just my problem  :smile_cat: so LGTM